src/scripts/llm.js

```diff
@@ -62,14 +83,13 @@ async function generateResponse(actor, input) {
     let response = tokenizer.decode(tokens[0], { skip_special_tokens: false });
     response = response.substring(prompt.length);
 
-    let prefixWithTalk = await actor.getFlag("unkenny", "prefixWithTalk") || false;
-    if (prefixWithTalk) {
-        response = "/talk " + response;
-    }
-
     await info.close();
 
     return response;
 }
 
+async function getAPIResponse(actor, input) {
```

Comment:
I think we can inline this function, just directly await generateResponseApi in line 31.

Reply:
Done.